File: CSharp/URI_Projects/1015_DistanciaEntreDoisPontos.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace URI_Projects
{
    class _1015_DistanciaEntreDoisPontos
    {
        private const int V = 0;

        public static void DistanciaEntreDoisPontos()
        {
            // Leia os quatro valores correspondentes aos eixos x e y de dois pontos quaisquer no plano, p1(x1,y1) e p2(x2,y2) e calcule a distância entre eles, mostrando 4 casas decimais após a vírgula, segundo a fórmula:
            string[] p1;
            string[] p2;

            p1 = Console.ReadLine().Split(" "); // armazenara x1 e y1
            p2 = Console.ReadLine().Split(" "); // armazenara x2 e y2


            double[] x1y1 = { double.Parse(p1[0]), double.Parse(p1[1]) };
            double[] x2y2 = { double.Parse(p2[0]), double.Parse(p2[1]) };

            double distancia = Math.Sqrt(((x2y2[0] - x1y1[0]) * (x2y2[0] - x1y1[0])) + ((x2y2[1] - x1y1[1]) * (x2y2[1] - x1y1[1])));

            Console.WriteLine(distancia.ToString("0.0000"));
        }
    }
}

File: JavaScript/Iniciante/older/1015_DistanciaEntreDoisPontos.js
/*
 * Leia os quatro valores correspondentes aos eixos x e y de dois pontos
 * quaisquer no plano, p1(x1,y1) e p2(x2,y2) e calcule a distância entre eles,
 * mostrando 4 casas decimais após a vírgula, segundo a fórmula:

 * Distancia = √((x2-x1)² + (y2-y1)²)

 * Entrada
 * O arquivo de entrada contém duas linhas de dados.
 * A primeira linha contém dois valores de ponto flutuante:
 * x1 y1 e a segunda linha contém dois valores de ponto flutuante x2 y2.

 * Saída
  Calcule e imprima o valor da distância segundo a fórmula fornecida,
  *  com 4 casas após o ponto decimal.
 *
 * trocar "prompt()" por "lines.shift()" quando for mandar pro URI
 */

function DistanciaEntreDoisPontos() {
  const texto1 = prompt();
  const texto2 = prompt();

  // const texto1 = "-2.5 0.4";
  // const texto2 = "12.1 7.3";

  const p1 = texto1.split(" ");
  const p2 = texto2.split(" ");

  const distancia = Math.sqrt((Math.pow((p2[0] - p1[0]), 2) + (Math.pow((p2[1] - p1[1]), 2))));
  console.log(distancia.toFixed(4));
}

DistanciaEntreDoisPontos();